packages/wallet-connector/jest.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+	roots: ["<rootDir>/src"],
+	setupFiles: ["<rootDir>/jest.setup.js"],
+	transform: {
+		"^.+\\.ts?$": "ts-jest",
+	},
+	transformIgnorePatterns: [
+		"<rootDir>/build/",
+		"<rootDir>/node_modules/",
+	],
+}
```

Comment:
Please, add the following property 
`	moduleNameMapper: {
		"source-map-support/register": "identity-obj-proxy",
	},` and install identity-obj-proxy package as dev dependency to avoid error lines mistakes